docs/k8s-quickstart.asciidoc

```diff
@@ -224,20 +224,20 @@ Use `kubectl port-forward` to access Kibana from your local workstation:
 kubectl port-forward service/quickstart-kb-http 5601
 ----
 +
-Open `https://localhost:5601` in your browser. Your browser will show a warning because the self-signed certificate configured by default is not verified by a third party certificate authority and not trusted by your browser. You can either configure a link:k8s-accessing-elastic-services.html#k8s-setting-up-your-own-certificate[valid certificate] or acknowledge the warning for the purposes of this quick start.
+Open `https://localhost:5601` in your browser. Your browser will show a warning because the self-signed certificate configured by default is not verified by a third party certificate authority and not trusted by your browser. You can temporarily acknowledge the warning for the purposes of this quick start but it is highly recommended that you <<{p}-setting-up-your-own-certificate,configure valid certificates>> for any production deployments.
 +
-Login with the `elastic` user. Retrieve its password with:
+Login as the `elastic` user. The password can be obtained with the following command:
 +
 [source,sh]
 ----
-echo $(kubectl get secret quickstart-es-elastic-user -o=jsonpath='{.data.elastic}' | base64 --decode)
+kubectl get secret quickstart-es-elastic-user -o=jsonpath='{.data.elastic}' | base64 --decode; echo
 ----
 
 [float]
 [id="{p}-upgrade-deployment"]
 === Upgrade your deployment
 
-You can apply any modification to the original cluster specification. The operator makes sure that your changes are applied to the existing cluster, while avoiding downtime.
+You can apply any modifications to the original cluster specification. The operator ensures that your changes are applied to the existing cluster with no disruption.
```

Comment:
I wasn't sure about this because both seemed "right" but "any modifications" sounded a little off to me. It looks like it's basically a free for all:
https://ell.stackexchange.com/questions/1155/should-nouns-after-any-be-singular-or-plural
and another example of English being terrible.

For the second part, there is a disruption in some sense since we will be restarting pods, so I wonder what the best way to express that is